dashboard/app/assets/stylesheets/DanceAiModal.scss

```diff
@@ -0,0 +1,105 @@
+// Fade in for 1 second after a 0 second delay.
+@keyframes appear0 {
```

Comment:
Feels like this could probably be a mixin? Something like `@include appearWithDelay($index)` which has a mapping of `index` to starting %? In fact it could even output the whole `animation: appear1 2s;` line so that you don't have to include the duration explicitly

Reply:
Oh, that was fun!